ca_tools/scripts/GTSys.py

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+import math
+
+import rospy
+from geometry_msgs.msg import Pose, Twist
+from nav_msgs.msg import Odometry
+
+
+class GTSys():
+
+    _pose = Pose()
+    _twist = Twist()
+
+    def __init__(self):
+        self._sub_gts=rospy.Subscriber('gts', Odometry, self._callback)
+
+    def _callback(self,data):
+        self._pose = data.pose.pose
+        self._twist = data.twist.twist
```

Comment:
Is `data.twist.twist` correct for `Odometry`? Please, double-check.

Reply:
Yes, it is. [Odometry](http://docs.ros.org/melodic/api/nav_msgs/html/msg/Odometry.html) has a TwistWithCovariance variable.